spec/models/users_organization_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UsersOrganization, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
;) habtm unless we have something here?

Reply:
@soey I thought you and I we both agreed that it would be the cleanest solution to have a join model between users and organization? I wouldn't be against starting with `has_many, through:`

Reply:
It's ok for me.

Reply:
@roschaefer: I think you are referring to the relationship between contributor and organization .. where we probably have to implement the note and mount the taggings  to